src/rime/lever/deployment_tasks.cc

```diff
@@ -616,7 +616,7 @@ bool CleanupTrash::Run(Deployer* deployer) {
 bool CleanOldLogFiles::Run(Deployer* deployer) {
   bool success = true;
 #ifdef RIME_ENABLE_LOGGING
-  if (FLAGS_logtostderr) {
+  if (FLAGS_log_dir.empty()) {
```

Comment:
should it be `FLAGS_logtostderr || FLAGS_log_dir.empty()`?

Reply:
No, we don't use `FLAGS_logtostderr` now, `google::LogToStderr()` will not make this flag be `true`.

Reply:
Indeed it's not used anywhere by a quick grep. But let's add it for the sake of completeness, if it is semantically correct.

Reply:
Get it, will be done soon.